table/persistent_cache_helper.cc

```diff
@@ -2,7 +2,9 @@
 //  This source code is licensed under the BSD-style license found in the
 //  LICENSE file in the root directory of this source tree. An additional grant
 //  of patent rights can be found in the PATENTS file in the same directory.
+
 #include "table/persistent_cache_helper.h"
+#include "table/block_based_table_reader.h"
```

Comment:
I tried many things until I reached this organization that works. It also seems cleaner to me compared to what we had before: persistent_cache_helper should not depend on  block_based_table_reader.